src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -156,7 +156,7 @@ public static class TimeSeriesCatalog
         /// <param name="outputColumnName">Name of the column resulting from data processing of <paramref name="inputColumnName"/>.
         /// The column data is a vector of <see cref="System.Double"/>. The length of this vector varies depending on <paramref name="detectMode"/>.</param>
         /// <param name="inputColumnName">Name of column to process. The column data must be <see cref="System.Double"/>.</param>
-        /// <param name="threshold">The threshold to determine anomaly, score larger than the threshold is considered as anomaly. Must be in [0,1]. Default value is 0.3.</param>
+        /// <param name="threshold">The threshold to determine anomaly. Scores larger than the threshold are considered as anomalies. This value must fall between [0,1]. Default value is 0.3.</param>
```

Comment:
I believe "score" here is the anomaly score calculated for each data point. It is the circled value here, where the anomaly score is calculated for each time-series chucks:
![anomaly_score](https://user-images.githubusercontent.com/5262061/82708169-77c63e00-9c32-11ea-80df-81396658696b.PNG)
[This ](https://arxiv.org/pdf/1906.03821.pdf)is the paper of this SRCNN anomaly detection model. As such I propose the following change:
```suggestion
        /// <param name="threshold">The threshold to determine an anomaly. An anomaly is detected when the calculated anomaly score for a given time-series chunk is more than the set threshold. This threshold must fall between [0,1], and its default value is 0.3.</param>
```